src/Commands/Setup.php

```diff
@@ -230,7 +234,10 @@ private function setupHelper(): void
         $replaces = [
             '$this->helpers = array_merge($this->helpers, [\'auth\', \'setting\']);' => $check,
         ];
-        $this->replace($file, $replaces);
+        $return = $this->replace($file, $replaces);
+        if ($return) {
```

Comment:
Since it is always a boolean can we skip the intermediary?
```suggestion
        if ($this->replace($file, $replaces)) {
```